test/test_casts.py

```diff
@@ -0,0 +1,19 @@
+import pytest
+
+from enum import Enum
+from typedconfig.casts import enum_cast
+
+
+class ExampleEnum(Enum):
+    RED = 1
+    GREEN = 2
+    BLUE = 3
+
+
+def test_enum_cast():
+    getter = enum_cast(ExampleEnum)
+
+    assert getter("BLUE") == ExampleEnum.BLUE
+
+    with pytest.raises(KeyError):
+        getter("PURPLE")
```

Comment:
Please could this be two tests? (e.g. `test_enum_cast_valid` and `test_enum_cast_invalid`)